onnx/defs/math/defs.cc

```diff
@@ -95,6 +95,11 @@ ONNX_OPERATOR_SET_SCHEMA(
     7,
     OpSchema().FillUsing(MathDocGenerator("subtraction")));
 
+ONNX_OPERATOR_SET_SCHEMA(
+    Mod,
+    10,
+    OpSchema().FillUsing(MathDocGenerator("modulus")));
```

Comment:
if we support both mod for int and fmod for floating numbers, we should make it clear here.

Reply:
There is no fmod operator suggested. Mod operator accepts both floating point and integers. The above shortcut points to the following line:
```
    schema.TypeConstraint(
        "T",
        OpSchema::numeric_types_for_math_reduction(),
        "Constrain input and output types to high-precision numeric tensors.");
```
and the numeric_types_for_match_reduction function is defined as below which covers the applicable cases in our scenario:

```
  static const std::vector<std::string>& numeric_types_for_math_reduction() {
    static const std::vector<std::string> numeric_types_for_math_reduction = {
        "tensor(uint32)",
        "tensor(uint64)",
        "tensor(int32)",
        "tensor(int64)",
        "tensor(float16)",
        "tensor(float)",
        "tensor(double)"};
    return numeric_types_for_math_reduction;
  }
```

Reply:
Actually I just noticed the diffs between fmod and mod. I will make sure Mod operator can emulate both.

Reply:
updated docs and examples